common/presenters/move-to-card-component.js

```diff
@@ -9,21 +9,47 @@ function moveToCardComponent({
   showMeta = true,
   showTags = true,
   showStatus = true,
+  showToLocation = false,
+  showFromLocation = false,
   hrefSuffix = '',
 } = {}) {
   return function item(move) {
-    const { id, reference, profile, status } = move
+    const {
+      id,
+      reference,
+      profile,
+      status,
+      to_location: toLocation,
+      from_location: fromLocation,
+    } = move
     const href = profile ? `/move/${id}${hrefSuffix}` : ''
     const excludedBadgeStatuses = ['cancelled']
 
     showTags = isCompact ? false : showTags
 
+    const moveMetaItems = []
+
+    if (showToLocation) {
```

Comment:
What should happen if `toLocation` or `toLocation.title` is undefined? Should it show a label with no text, or not show it at all?

Reply:
In those scenarios, it won't show the meta. 

The [`profileToCardComponent`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/presenters/profile-to-card-component.js#L54) will remove any meta that doesn't have a value:

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/2e1fd309d03239e7b30cd14c1fa2bfd21621acb0/common/presenters/profile-to-card-component.js#L54

However, there is also [a helper](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/2e1fd309d03239e7b30cd14c1fa2bfd21621acb0/common/helpers/move/set-missing-to-location.js#L3) to populate moves with the correct to location if it doesn't exist - for example Prison recall moves.